examples/ex29.cpp

```diff
@@ -0,0 +1,375 @@
+//                                MFEM Example 29
+//
+// Compile with: make ex29
+//
+// Sample runs:  ex29
+//               ex29 -r 2 -sc
+//               ex29 -mt 3 -o 4 -sc
+//               ex29 -mt 3 -r 2 -o 4 -sc
+//
+// Description:  This example code demonstrates the use of MFEM to define a
+//               finite element discretization of a PDE on a 2 dimensional
+//               surface embedded in a 3 dimensional domain. In this case we
+//               solve the Laplace problem -Div(sigma Grad u) = 1, with
+//               homogeneous Dirichlet boundary conditions, where sigma
+//               is an anisotropic diffusion constant defined as a 3x3 matrix
+//               coefficient.
+//
+//               This example demonstrates the use of finite element
+//               integrators on 2D domains with 3D coefficients.
+//
+//               We recommend viewing example 1 before viewing this example.
```

Comment:
This example is also very similar to example 7 and we should refer to that example as well.